data/test_assets/dataset_tests/scenes/dataset_test_scene.scene_instance.json

```diff
@@ -1,19 +1,24 @@
 {
+	"translation_origin":"com",
 	"stage_instance":{
+		"translation_origin":"asset_local",
 		"template_name":"modified_test_stage",
 		"translation":[1.1,2.2,3.3]
 	},
 	"object_instances":[
 	{
+		"translation_origin":"asset_local",
 		"template_name":"dataset_test_object1",
 		"translation":[0.1,0.2,0.3]
 	},
 	{
+		"translation_origin":"asset_local",
 		"template_name":"dataset_test_object2",
 		"translation":[0.3,0.4,0.5]
 	}
 	],
 	"default_lighting":"modified_test_lights",
+	"frustum_culling":true,
```

Comment:
I think we decided to remove this from config system and instead just keep SimulatorConfiguration as the entry point.